splunklogger.js

```diff
@@ -360,8 +493,34 @@ SplunkLogger.prototype.use = function(middleware) {
 };
 
 /**
+ * Estimates the size in bytes of the events in the
+ * <code>queue</code> parameter as if they were sent
+ * in a single HTTP request.
+ *
+ * @returns {number} the estimated size in bytes.
+ */
+SplunkLogger.prototype.calculateBatchSize = function() {
+    var size = 0;
+    for (var i = 0; i < this.eventSizes.length; i++) {
```

Comment:
This will go away with some refactoring, I will use a running count like Java/.Net